benchopt/util.py

```diff
@@ -290,28 +287,28 @@ def is_included(name, include_patterns=None):
     return False
 
 
-def create_venv(env_name, recreate=False):
-    """Create a virtual env with name env_name and install basic utilities"""
+def create_condaenv(env_name, recreate=False):
+    """Create a conda env with name env_name and install basic utilities"""
 
-    env_dir = f"{VENV_DIR}/{env_name}"
+    force = " -y" * recreate
 
-    if not os.path.exists(env_dir) or recreate:
-        print(f"Creating venv {env_name}:...", end='', flush=True)
-        venv.create(env_dir, with_pip=True)
-        # Install benchopt as well as packages used as utilities to install
-        # other packages. The install of benchopt is done with the -e flag
-        # to ease development process
-        # XXX: add a flag to enable/disable develop install?
-        pip_install_in_env("numpy", "cython", "-e .", env_name=env_name)
-        print(" done")
+    print(f"Creating conda env {env_name}:...", end='', flush=True)
+    subprocess.run(f"conda create{force} -n {env_name}", shell=True)
```

Comment:
As we don't care about extra spaces, I would rather have `conda create {force} -n ...`, it is less error prone.

Reply:
You should catch the exit code and raise a proper error if the command fail.

Reply:
Everytime we run subprocess.run() ? Isn't this too heavy ?